metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -714,17 +725,19 @@ def get_workunits(self) -> Iterable[Union[MetadataWorkUnit, SqlWorkUnit]]:
             profile_requests: List["GEProfilerRequest"] = []
             if sql_config.profiling.enabled:
                 profiler = self.get_profiler_instance(inspector)
-
+            print("Entering DB Name")
             db_name = self.get_db_name(inspector)
+            print("DB NAME")
+            print(db_name)
```

Comment:
Can you use the logger instead of print statements? It looks like `logger.debug` might be most appropriate

Reply:
Ah those were for testing. I'll remove those.